agenta-backend/agenta_backend/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+ env =
+     DATABASE_MODE=test_mode
```

Comment:
Please change the DATABASE_MODE from 'test_mode' to 'test' as the test cases are currently running using the v2 database. Once done, remove line 11 in `agenta-backend/agenta_backend/config.toml`, followed by line 14 and 15 in `agenta-backend/agenta_backend/db_engine.py`. Finally, modify line 25 to use the database mode from the os environ.

After completing this task, please run the tests to determine if it passes.

Reply:
Okay

Reply:
There's no `db_engine.py` file in the project.

Reply:
> Please change the DATABASE_MODE from `test_mode` to `test` as the test cases are currently running using the `v2` database. Once done, remove line 11 in `agenta-backend/agenta_backend/config.toml`, followed by line 14 and 15 in `agenta-backend/agenta_backend/db_engine.py`. Finally, modify line 25 to use the database mode from the os environ.
> 
> After completing this task, please run the tests to determine if it passes.

There's no `db_engine.py` file in the project.

Reply:
Yes, there is. Kindly check `agenta-backend/agenta_backend/models/db_engine.py`:

![image](https://github.com/Agenta-AI/agenta/assets/55067204/a8ba0ac1-cb32-43d2-8a10-23e2622192bd)